mailbox/backup/src/main/java/org/apache/james/mailbox/backup/Zipper.java

```diff
@@ -74,6 +82,27 @@ private void storeInArchive(Mailbox mailbox, ZipArchiveOutputStream archiveOutpu
 
         archiveOutputStream.putArchiveEntry(archiveEntry);
         archiveOutputStream.closeArchiveEntry();
+
+        if (!annotations.isEmpty()) {
+            String annotationsDirectoryPath = name + "/" + ANNOTATION_DIRECTORY;
+            ZipArchiveEntry annotationDirectory = (ZipArchiveEntry) archiveOutputStream.createArchiveEntry(
+                new Directory(annotationsDirectoryPath), annotationsDirectoryPath);
+            archiveOutputStream.putArchiveEntry(annotationDirectory);
+            archiveOutputStream.closeArchiveEntry();
+            annotations.forEach(Throwing.consumer(annotation ->
+                storeInArchive(annotation, annotationsDirectoryPath, archiveOutputStream)));
+        }
+    }
+
+    private void storeInArchive(MailboxAnnotation annotation, String directory, ZipArchiveOutputStream archiveOutputStream) throws IOException {
+        String entryId = directory + "/" + annotation.getKey().asString();
+        ZipArchiveEntry archiveEntry = (ZipArchiveEntry) archiveOutputStream.createArchiveEntry(new File(entryId), entryId);
+
+        archiveOutputStream.putArchiveEntry(archiveEntry);
+        IOUtils.copy(IOUtils.toInputStream(annotation.getValue().orElse(""), Charsets.UTF_8
```

Comment:
a Printer seems more appropriate

Reply:
Do you mean a PrintWriter ? If not could you indicate me which class to use ?

Reply:
I think I prefer, the IOUtils.copy solution because according to the RFC, the annotation could be a string or binary data. And the Printwriter will not be adequate if we decide to support binary data in the annotations.

Reply:
Yes, I was talking about PrintWriter, you are right.
Your argument about binary doesn't hold as annotation.getValue returns a String anyway.